explain more things (#447)
TYPO3-Documentation/TYPO3CMS-Book-ExtbaseFluid: Documentation/3-BlogExample/9-Rendering-the-output-with-fluid.rst

All the XML tags with namespace »f« stand out, like `<f:for>` or `<f:link.action>`.
These tags are provided by Fluid and represent different functionalities.

* `<f:format.nl2br>[…]</f:format.nl2br>` :
   modifies linebreaks (new lines) to `<br />` tags.
* `<f:link.action action="edit">` :
  creates a link tag that links to the :php:`editAction()` of the current controller.

> * `<f:for each="{paginator.paginatedItems}" as="blog">[...]</f:for>` :

Reviewer: ```suggestion
* `<f:for each="{paginator.paginatedItems}" as="blog">[…]</f:for>` :
```

   iterates over the paginated Blog objects found in Blogs.

> * `<f:link.action action="index" controller="Post" arguments="{blog : blog}">{blog.title} ({f:translate(key: 'blog.numberOfPosts', arguments: '{numberOfPosts: \'{blog.posts -> f:count()}\'}')})</f:link.action>` :

Reviewer: ```suggestion
* `<f:link.action action="index" controller="Post" arguments="{blog : blog}">[…]</f:link.action>` :
```

Author: All those parts are necessary for the following explanations.

Reviewer: I know, that's why I've split this into 2 bullets.

>    creates a link to the `indexAction` method of the `PostController` which is `public function indexAction(Blog $blog, $tag = null)`. 

Reviewer: ```suggestion
   creates a link to the :php:`indexAction` method of the :php:`PostController` which is :php:`public function indexAction(Blog $blog, $tag = null)`. 
```

>    The first parameter blog is passed to this method. A temporary variable `numberOfPosts` is filled by the `f:count()` method.
>    The count method gives the numbers of posts. The text key `blog.numberOfPosts`of the file 
>    :file:`Resources/Private/Language/locallang.xml` refers to the text `%d posts`. 

Reviewer: ```suggestion
   The blog stored in the variable `{blog}` is passed as `blog` parameter to the action.

* `{f:translate(key: 'blog.numberOfPosts', arguments: '{numberOfPosts: \'{blog.posts -> f:count()}\'}')}`
   The translation key `blog.numberOfPosts` refers to the translation file:`Resources/Private/Language/locallang.xml`.
   The number of `{blog.posts}` is counted using the `f:count()` viewhelper and passed as argument to the `f:translate()` viewhelper. This viewhelper inserts the argument into the translation text `%d posts`. 
```

>    The fluid method `f:translate(...)` fetches this string and replaces its `%d` argument by the number of posts for the current blog.

Reviewer: ```suggestion
```

>    `{blog.posts}` is the array of posts which belong to the current blog. The class `Blog` contains the protected variable `$posts`.

Reviewer: ```suggestion
   `{blog.posts}` is the list of posts which belong to the current blog. The class `Blog` contains a getter method `getPosts()` which is automatically used to access the list of posts (:php:`\TYPO3\CMS\Extbase\Persistence\ObjectStorage`).
```

>    This is the member variable (of type `\TYPO3\CMS\Extbase\Persistence\ObjectStorage`) where the posts are stored, e.g. in `function addPost(Post $post)` .

Reviewer: ```suggestion
```

All the XML tags with namespace »f« stand out, like `<f:for>` or `<f:link.action>`.
These tags are provided by Fluid and represent different functionalities.

*  `<f:format.nl2br>[…]</f:format.nl2br>` :
   modifies linebreaks (new lines) to `<br />` tags.
*  `<f:link.action action="edit">` :
   creates a link tag that links to the :php:`editAction()` of the current controller.
*  `<f:for each="{paginator.paginatedItems}" as="blog">[…]</f:for>` :
   iterates over the paginated Blog objects found in Blogs.
*  `<f:link.action action="index" controller="Post" arguments="{blog : blog}">[…]</f:link.action>` :
   creates a link to the :php:`indexAction` method of the :php:`PostController` which is :php:`public function indexAction(Blog $blog, $tag = null)`. 
   The blog stored in the variable `{blog}` is passed as `blog` parameter to the action.
*  `{f:translate(key: 'blog.numberOfPosts', arguments: '{numberOfPosts: \'{blog.posts -> f:count()}\'}')}`
   The translation key `blog.numberOfPosts` refers to the translation 

>    file:`Resources/Private/Language/locallang.xml`. The number of `{blog.posts}` is counted using the 

Reviewer: ```suggestion
   file:`Resources/Private/Language/locallang.xlf`. The number of `{blog.posts}` is counted using the 
```
